core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.template.html

```diff
@@ -50,7 +50,7 @@ <h3>Upload Thumbnail</h3>
             <i class="material-icons oppia-vcenter">&#xE14C;</i>
           </button>
           <div>
-            <img ng-if="uploadedImage" class="oppia-thumbnail-image" ng-src="<[uploadedImage]>" height="<[aspectRatio === '4:3' ? '186px' : '180px']>" width="<[aspectRatio === '4:3' ? '248px' : '320px']>">
+            <oppia-thumbnail-display ng-if="uploadedImage" [classes]="['oppia-thumbnail-image']" [img-src]="uploadedImage" [aspect-ratio]="aspectRatio" [background]="bgColor"></oppia-thumbnail-display>
```

Comment:
Just noting that a screenshot/video of this should be shown under the "proof that changes are correct" section in the PR template, since this PR changes how the background colour is set.

Reply:
This is a part of #11306.
See https://github.com/oppia/oppia/pull/11850#issuecomment-786942336 for details.